akka-docs/rst/java/remoting-artery.rst

```diff
@@ -134,8 +138,8 @@ Looking up Remote Actors
 
 ``actorSelection(path)`` will obtain an ``ActorSelection`` to an Actor on a remote node, e.g.::
 
-  val selection =
-    context.actorSelection("akka://actorSystemName@10.0.0.1:25520/user/actorName")
+  ActorSelection selection =
+    context.actorSelection("akka.tcp://app@10.0.0.1:2552/user/serviceA/worker");
```

Comment:
No `tcp` when using artery and I think we tried to keep 25520 as the port in the examples, so please just keep the original path.